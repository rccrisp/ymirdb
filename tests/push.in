set a 1 2 3
push a 0
set b -1 -2 -3
push a b
min a
sum a
backward b
forward a
push a b b b
len a
forward a
backward b
get a
bye