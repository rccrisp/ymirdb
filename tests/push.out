> ok

> ok

> ok

> ok

> -3

> 0

> a

> b

> ok

> 16

> b

> a

> [b b b b 0 1 2 3]

> bye
